src/web/client/test/integration/WebExtensionContext.test.ts

```diff
@@ -0,0 +1,784 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can we have small PRs? 
Its a huge PR to be reviewed

Reply:
I can remove he Package.Json from here because my one of the PR is going to merge in main which have all package.json changes